cpucounters.cpp

```diff
@@ -188,7 +191,7 @@ class InstanceLock
 
 class TemporalThreadAffinity  // speedup trick for Linux
```

Comment:
Should fix this comment "speedup trick for Linux" while you are here - i.e. add FreeBSD/DragonFlyBSD.